.github/workflows/budibase_ci.yml

```diff
@@ -160,31 +160,6 @@ jobs:
             yarn test --scope=@budibase/server
           fi
 
-  test-pro:
```

Comment:
This was added in the past as pro test was really flaky, not really related to this PR